pkg/kv/kvserver/allocator/allocatorimpl/allocator_scorer.go

```diff
@@ -70,26 +70,16 @@ const (
 	// away from the mean.
 	minRangeRebalanceThreshold = 2
 
-	// MaxL0SublevelThreshold is the number of L0 sub-levels of a store
-	// descriptor, that when greater than this value and in excees of the
-	// average L0 sub-levels in the cluster - will have the action defined by
-	// l0SublevelsThresholdEnforce taken. This value does not affect the
-	// allocator in deciding to remove replicas from it's store, only
+	// DefaultIOOverloadThreshold is the IO overload score of a store
+	// descriptor, that when greater than this value and in excess of the
+	// average IO overload of comparable candidates - will have the action
+	// defined by StoreHealthEnforcement taken. This value does not affect
+	// the allocator in deciding to remove replicas from it's store, only
```

Comment:
[nit] Typo: it's -> its. You could simplify the language structure: "DefaultIOOverloadThreshold is the IO overload score of a store descriptor, that when greater than this value and in excess of the average IO overload of comparable candidates - will have the action defined by StoreHealthEnforcement taken." to "DefaultIOOverloadThreshold is used to avoid allocating to stores with an IO overload score greater than what's set. This is typically used in conjunction with IOOverloadWaterMark below.". Also, "WaterMark" doesn't sound right -- it's threshold above the mean, like your comment below suggests.

Reply:
Updated the comment. Also updated IOOverloadWatermark to be IOOverloadMeanThreshold